packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -72,7 +77,15 @@ class ${class_name}:
         if grpc_channel is not None:
             self._stub = v2_measurement_service_pb2_grpc.MeasurementServiceStub(grpc_channel)
         self._create_file_descriptor()
-        
+        self.pin_map_id: str = ""
+        self.sites: list[int] = [0]
```

Comment:
Use `List` for type-hinting.
```suggestion
        self.sites: List[int] = [0]
```

Reply:
Introduced a setter for sites, which strictly expects the datatype to be `List[int]`. So, removing the explicit datatype mention here.